src/logic.js

```diff
@@ -141,4 +141,8 @@ function move(id) {
     return false;
   }
   return 0;
-}
\ No newline at end of file
+};
+
+export {
+  start, cancel, playAgain, gameBoard, validMove, game, engine, gameEnd, symbol, check, player,
```

Comment:
You should have tests for each of these listed here!

Reply:
i.e 
```js
describe('start', () => {
// test whatver etc.
});
```
You may even want to test victory.

Ideally, your project was to be set up into 3 different modules/factories originally (player, gameboard, game) and Ideally you'd test functions within these but your setup is a bit confusing so I'd suggest you think of your project like this.

You are to test all your functions within these objects. However, in your case, since there is no real separation you'd have to test all functions here in logic